frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "bootstrap": "^4.5.3",
     "bootstrap-social": "^5.1.1",
     "font-awesome": "^4.7.0",
+    "joi": "^17.4.0",
```

Comment:
I believe we don't need it. Can you please remove it?

Reply:
Sure